pkg/telemetry/applicationinsights.go

```diff
@@ -169,7 +167,7 @@ func (p *ApplicationInsights) GetHTTPMiddleware() (func(handler http.Handler) ht
 			trace.Tags.Operation().SetCorrelationVector(request.Header.Get("correlation-context"))
 			parentID := request.Header.Get("request-id")
 			trace.Tags.Operation().SetParentId(parentID)
-			trace.Tags.Operation().SetId(parentID + uid)
+			trace.Tags.Operation().SetId(parentID)
```

Comment:
Removing the uid from the parentID will break the correlation logic. But due to the fact that we want to remove application insights anyway I would suggest to disable application insights for now and remove application insights in a later issue.